### PR TITLE
Make evil-collection-mode-list customizable.

### PR DESCRIPTION
**Pull request by dieggsy**
Would be nice if the user could use the custom interface to determine what gets enabled and what doesn't.

### CODE REVIEW

**Comment by dieggsy** [comment]
Well, looks like this needs a bit more work for the custom interface to display the list properly

**Comment by dieggsy** [comment]
Force pushed some changes, though `sexp` is probably still too broad

File: evil-collection.el [hunk 1]
```diff
@@ -34,7 +34,11 @@
 ;;      (evil-minibuffer-init)
 
 ;;; Code:
-(defvar evil-collection-mode-list
+(defgroup evil-collection nil
+  "A set of keybindings for Evil mode"
+  :group 'evil)
```
Comment (jojojames) [inline evil-collection.el]: Group is probably 'evil-collection or 'evilcoll (after I make the namespace changes).

**Comment by jojojames** [comment]
This looks reasonable otherwise. Once the :group variable is changed, I can take a look after merging.

**Comment by dieggsy** [comment]
The group is correct for the `defcustom` hovever the `:group` in a defgroup should not be itself - I think this leads to the group containing itself

**Comment by noctuid** [comment]
Yeah, `:group` is correct for the `defgroup`. The `:prefix` would be `evilcoll-` or  whatever ends up being used.

**Comment by jojojames** [comment]
Cool, sounds good. :)

I'll merge this and test it later.

**Pull request merged**